### DISCUSSION TITLE
Global Data to All Team in Multi Tenancy

### DISCUSSION THREAD

**Discussion by decopiller** [filamentphp/filament#8586] [Help]
I have table permissions, now I have to add foreign key team_id to the table because of requirement multi tenancy. How to make permissions table available to all team, and remove foreign key team_id?

**Comment by leandrocfe** [comment]
Is the Permission a resource? What is the issue when you remove team_id field?

**Reply by decopiller** [reply]
I think we can use `scopeEloquentQueryToTenant `in resource class

**Comment by decopiller** [comment]
Yes, it is a resource.

If i remove the team_id, it show `Unknown column permissions.team_id`.

Is it should add team_id in every tables when I use multitenancy in filament? Or can I choose some tables which has to applied with tenancy? I hope you understand my words
